alf/algorithms/sac_algorithm.py

```diff
@@ -145,6 +145,8 @@ def __init__(self,
                  config: TrainerConfig = None,
                  critic_loss_ctor=None,
                  target_entropy=None,
+                 prior_actor_ctor=None,
+                 target_kld_per_dim=3.,
```

Comment:
May be useful to add a note that target_kld_per_dim  will be used to update self._target_entropy value when prior_actor is used.